src/Participation/LotteryFilter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace PhpAb\Participation;
+
+class LotteryFilter implements FilterInterface
+{
+    /**
+     * @float
+     */
+    private $propability;
+
+    /**
+     * @param int $propability The probability for the lottery in percent
```

Comment:
Yes i first wanted to normalize beween 0 and 1 but than i have to multiply it by 100 and round it somehow to comply with mt_rand accepting int only. So i decided to use int directly.

are you ok with that?